lib/ttfunk/table/maxp.rb

```diff
@@ -50,6 +52,10 @@ def parse!
             @max_storage, @max_function_defs, @max_instruction_defs,
             @max_stack_elements, @max_size_of_instructions,
             @max_component_elements, @max_component_depth = read(28, 'Nn*')
+
+          # a number of fonts omit these last two bytes for some reason,
```

Comment:
I believe so, yes. I saw the `nil` issue in fonts where the maxp table was the last table in the font. We do have the length here so it would not be too difficult to protect against reading past the end of the table.

Reply:
Let's do that.

Reply:
Done (and found a bug as a result!)